### DISCUSSION TITLE
Allow simple changing of /rss and /json URL endpoints

### DISCUSSION THREAD

**Discussion by hailspuds** [microfeed/microfeed#76] [Ideas] [answered]
It'd be great to be able to change the URL for the RSS and JSON feeds from the default /rss/ and /json/ to anything you want.

I need to serve the RSS feed at a different URL.

**Comment by wenbinf** [comment, marked as answer]
great suggestion!

for now, you can use Cloudflare's Page Rules to redirect any path to other paths:

https://developers.cloudflare.com/support/page-rules/configuring-url-forwarding-or-redirects-with-page-rules/
